flent/scripts/ss_iterate.sh

```diff
@@ -18,31 +18,39 @@ count=10
 interval=0.1
 host=localhost
 filter=""
+surveil=""
 
 usage()
 {
-    echo "$0 -c count -I interval -H host -t target"
+    echo "$0 -c count -I interval -H host -t target -f filter"
```

Comment:
Hmm, instead of adding yet another arg here, maybe just get rid of -t and move the "dst $TARGET" into the Python code as part of the filter?